woocommerce/woocommerce
# Improve core functions test coverage

### All Submissions:






### Changes proposed in this Pull Request:


This PR adds unit tests for core function that currently do not have any tests.

### How to test the changes in this Pull Request:

1. Run `vendor/bin/phpunit tests/unit-tests/util/class-wc-tests-core-functions.php`
2. Ensure all the tests pass

### Other information:
